src/gui/rss/automatedrssdownloader.cpp

```diff
@@ -599,9 +597,11 @@ void AutomatedRssDownloader::updateFieldsToolTips(bool regex)
               + "<li>" + tr("Whitespaces count as AND operators (all words, any order)") + "</li>"
               + "<li>" + tr("| is used as OR operator") + "</li></ul></p>"
               + "<p>" + tr("If word order is important use * instead of whitespace.") + "</p>";
-        ui->lineContains->setToolTip(tip);
-        ui->lineNotContains->setToolTip(tip);
     }
+
+    tip += "<p>" + tr("An expression with an empty ") + "<tt>|</tt>" + tr(" clause (e.g. ") + "<tt>" + tr("expr|") + "</tt>" + tr(")");
```

Comment:
Translators will get confused here on what is being said/written. So IMO, let's add some disambiguation text.
```c++
"<p>" + tr("An expression with an empty %1 clause (e.g. %2)", "We talk about regex/wildcards in the RSS filters section here. So a valid sentence would be: An expression with an empty | clause (e.g. expr|)").arg("<tt>|</tt>").arg("<tt>expr|</tt>");
```